### PR TITLE
0.35.0 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software/release-notes.md [hunk 1]
```diff
@@ -26,6 +26,60 @@ If you're upgrading to receive a specific change, ensure the release note for th
 
 :::
 
+## 0.35.0 
+
+Release date: May 14, 2024
```
Comment (lzdanski) [inline software/release-notes.md:31]:
```suggestion
Release date: June 14, 2024
```

File: software/release-notes.md [hunk 2]
```diff
@@ -26,6 +26,60 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
+
+### Use deploy rollbacks to run previous versions of your code
+
+Deploy rollbacks are a new way to ensure that your Deployments can keep running after a broken code deploy or upgrade. When you trigger a rollback to a past deploy, your previous image and/or DAG code are redeployed and start running. 
+
+To configure deploy rollbacks, see [LIKNK]
```
Comment (lzdanski) [inline software/release-notes.md:37]: to add after merge to release branch

File: software/release-notes.md [hunk 3]
```diff
@@ -26,6 +26,60 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
+
+```sh
+kubectl create secret generic astronomer-bootstrap \
+  --from-literal connection="jdbc:mysql://host1:33060/mydb" \
+  --namespace astronomer
+```
+
+### Create Teams without using an IdP
+
+Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.
```
Comment (lzdanski) [inline software/release-notes.md:51]: adding link after merge into base branch

File: software/release-notes.md (hunk 1 above)
Comment (rujhan-arora-astronomer) [reply software/release-notes.md:31]: We are not releasing it on June 14. Probably second last or last week of june.

**Review by danielhoherd — COMMENTED** [review]
Looks mostly good. I'd like @Simpcyclassy @pgvishnuram or @rujhan-arora-astronomer to double-check the list of bug fixes since a LOT of them were houston related and because I've been on PTO for over a week and my memory of this release is a bit hazy.

File: software/release-notes.md (hunk 3 above)
Comment (tonyHuinker) [reply software/release-notes.md:51]: I would change this

From

"Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps."

to

"Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps."

**Comment by Simpcyclassy** [comment]
We need to add the relevant config
 
#### Enable rollback feature flag
```
global:
    deployRollbackEnabled
```

#### Enable cleanup feature flag
```
houston:
    cleanupDeployRevisions:
        enabled:  true
```

#### Configure which data to clean up
```
houston:
    cleanupDeployRevisions:
        olderThan:  90
```

File: software/release-notes.md [hunk 4]
```diff
@@ -26,6 +26,77 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
```
Comment (Simpcyclassy) [inline software/release-notes.md:51]: Updating comment as a suggestion
```suggestion
Astronomer [Teams](import-idp-groups.md) ccan now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps.
```
```suggestion
Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.
```

Comment (Simpcyclassy) [inline software/release-notes.md:51]: Updating comment as a suggestion

```suggestion
Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps.
```

File: software/release-notes.md [hunk 5]
```diff
@@ -26,6 +26,77 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
+
+### Create Teams without using an IdP
+
+Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps.
+
+### Additional improvements
+
+- DAG-only deploys are now fully compatible with OpenShift clusters.
+- You can now set a `priorityClass` to pods created by `templates/trust-private-ca-on-all-nodes/containerd-daemonset.yaml`. This allows you to set a high priority class for certain pods to force an evication for important processes. <!--https://github.com/astronomer/issues/issues/6366-->
+
```
Comment (lzdanski) [inline software/release-notes.md:57]:
```suggestion
- You can now set a `priorityClass` to pods created by `templates/trust-private-ca-on-all-nodes/containerd-daemonset.yaml`. This allows you to set a high priority class for certain pods to force an evication for important processes. <!--https://github.com/astronomer/issues/issues/6366-->
```

**Pull request merged**